sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -759,6 +761,7 @@ SNAPSHOT: 'SNAPSHOT';
 READ: 'READ';
 WRITE: 'WRITE';
 ONLY: 'ONLY';
+MACRO: 'MACRO';
```

Comment:
Just wanna confirm. I thought MACRO is a reseved keyword. Is it right? https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL